Expensify/App
# Make login form accessible to Password Managers

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/3855
$ https://github.com/Expensify/App/issues/5956

### Tests | QA Steps
1. Navigate to NewDot on the web.
2. Right-click on the email field.
3. Click on the 1Password option to show the autofill.
4. 1Password Autofill option should be shown on the input.

### Tested On


### Screenshots


#### Web

Updating Shortly...

#### Mobile Web


#### Desktop


#### iOS


#### Android
